packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -177,6 +177,14 @@ export interface ApplicationLoadBalancedServiceBaseProps {
    * @default false
    */
   readonly redirectHTTP?: boolean;
+
+  /**
+   * Specifies whether the Route53 record should be a CNAME or an A record using the Alias feature.
+   * This is useful if you need to work with DNS systems that do not support alias records.
```

Comment:
I think it would be better to create an issue to describe why we need this feature and link this PR to that issue.

The PR LGTM from code perspective.  I just don't understand the scenario. Will be great to elaborate it in the issue.

Reply:
copy @iamhopaul123

Reply:
Yeah agree with @pahud, could you create an issue elaborating the use case that this PR mainly addresses? It would also increase visibility for the other users with similar problem.